modules/openxAnalyticsAdapter.js

```diff
@@ -1141,14 +1136,58 @@ function buildAuctionPayload(auction) {
               ts,
               rendered,
               renderTime,
-              meta,
+              meta
             }
           })
         }
-      })
+      });
+
+      return {
+        code: adUnitCode,
+        incrementalLiftMicroCpmUSD: calculateIncrementalLift(bidRequests),
+        bidRequests
+      };
+
+      function buildUserIdPayload(userId) {
+        return utils._map(userId, (id, module) => {
+          return {
+            module: module,
+            id: getUserId(module, id)
+          };
+        })
+          .filter(({id}) => id)
+      }
     }).flat();
   }
 
+  function calculateIncrementalLift(bidRequests) {
+    let bidResponses = bidRequests.flatMap(bidRequest => bidRequest.bidResponses);
+    let [winningBidResponses, losingBidResponses] = partitionByWinningResponses(bidResponses);
+    let loserWithHigestCpm;
```

Comment:
are they all guaranteed to be in USD ?

Reply:
also what happens if the second price comes from the same bidder ? or is it guaranteed that a bidder will only have one response ?

Reply:
oh good point.  I'll leave it to you to do it :). I'll leave out the property in the payload.